src/styles/style.scss

```diff
@@ -0,0 +1,115 @@
+@import url(../styles/stars.scss);
+
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Roboto, sans-serif;
+}
+
+.card {
+  width: 198px;
+  border-radius: 5px;
+  border: 1px solid #F3F3F3;
+  background: #FFF;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin: 32px 19px 36px;
+}
+
+.card__title {
+  color: #060B35;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  line-height: 18px;
+  margin: 0 16px 4px;
+}
+
+.title {
+  color: #616070;
+  font-size: 10px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 14px;
+  margin: 0 16px;
+}
+
+.review {
+  display: flex;
+  width: 166px;
```

Comment:
Consider specifying paddings for the element container, to avoid specifying the width in every element